assets/scss/_util.scss

```diff
@@ -0,0 +1,60 @@
+//
```

Comment:
This is just a repeat from this PR just to get things compiling 
https://github.com/meetup/meetup-web-components/pull/32